gnome/gobject-introspection/Portfile

```diff
@@ -55,6 +55,12 @@ depends_lib-append \
 patchfiles-append   patch-fix-rpath-gir-typelib.diff
 patchfiles-append   patch-fix-scanner-in-build-execution.diff
 patchfiles-append   patch-fix-tools-python.diff
+# required for updated py-setuptools tp ver 74.0.0
+# Patch needs to be removed when upgraded version of introspection is used.
+# Except one rule of this patch will have to be made with new patch :
+# change runtime_path_envvar = ['LD_LIBRARY_PATH', 'DYLD_FALLBACK_LIBRARY_PATH'] if not lib_dirs_envvar else [lib_dirs_envvar]
+# To runtime_path_envvar = ['DYLD_LIBRARY_PATH'] if not lib_dirs_envvar else [lib_dirs_envvar]
```

Comment:
please add these comments in the patchfile if needed at all. Also, since the patch is an upstream fix - add the upstream URL in the patchfile